packages/eas-cli/src/commands/metadata/lint.ts

```diff
@@ -0,0 +1,69 @@
+import { getConfig } from '@expo/config';
+import { Flags } from '@oclif/core';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log from '../../log';
+import { loadConfigAsync } from '../../metadata/config/resolve';
+import { createMetadataContextAsync } from '../../metadata/context';
+import { MetadataValidationError, logMetadataValidationError } from '../../metadata/errors';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+
+export default class MetadataLint extends EasCommand {
+  static override description = 'validate the local store configuration';
+
+  static override flags = {
+    json: Flags.boolean({
+      description: 'Enable JSON output, non-JSON messages will be printed to stderr',
+      default: false,
+    }),
+    profile: Flags.string({
+      description:
+        'Name of the submit profile from eas.json. Defaults to "production" if defined in eas.json.',
+    }),
+  };
+
+  static override contextDefinition = {
+    ...this.ContextOptions.ProjectDir,
+  };
+
+  async runAsync(): Promise<void> {
+    Log.warn('EAS Metadata is in beta and subject to breaking changes.');
+
+    const { flags } = await this.parse(MetadataLint);
+    const { projectDir } = await this.getContextAsync(MetadataLint, {
+      nonInteractive: false,
```

Comment:
optional: this command looks fairly non-interactive so this could maybe be set to true?

Reply:
Yes, we are blocked on setting non-interactive to true for both `eas metadata:pull` and `eas metadata:push`, caused by the `credentials/ios/appstore/AppStoreApi.ts` (used in the `CredentialsContext`) using methods that could prompt users.

But because the `eas metadata:lint` command doesn't use the credentials part, we can set this to non interactive.